frontend/workflows/projectSelector/src/dash.tsx

```diff
@@ -33,7 +40,143 @@ const Dash = ({ children }) => {
         <DashStateContext.Provider value={state}>
           <ProjectSelector />
           <Box display="flex" flex={1}>
-            {children}
+            <BigGrid
```

Comment:
won't merge changes in this file. this is just in case PR reviewers want to test locally